.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         runs-on: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python-version: ["3.10", "3.11"]
+        python-version: ["3.10", "3.11"] # add 3.12 when p4p #145 is fixed
```

Comment:
Already being tracked by an issue, let's keep this neat

Reply:
I think this got here because of rebase was not possible directly, so I did some merge?